zh/backup-restore-snapshot-perf.md

```diff
@@ -84,19 +84,19 @@ EBS 卷快照恢复阶段包含以下阶段，详细信息，参考[基于 EBS 
 
 | 恢复阶段     | 恢复大致耗时 | 恢复总占比 | 备注                                                                            |
 | :--------: | :---------: | :------: | :----------------------------------------------------------------------------: |
-| 创建集群     | 30s         | 1%      | 包含下载 docker image 和启动 PD 的时间                                                 |
-| 卷恢复      | 20s         | 1%     | 包含启动 BR pod 和卷恢复的时间                                                         |
-| 启动 TiKV   | 2～30 min    | 58%      | TiKV 启动时长受卷性能影响，一般 3 分钟左右，当恢复数据包含事务时，此阶段耗时可能达到 30 分钟 |
-| 数据恢复阶段 | 2～20 min    | 38%       | 包含恢复 raft 共识层数据以及删除 MVCC 数据的时间                                      |
-| 启动 TiDB   | 1 min        | 2%       | 包含下载 docker image 和启动 TiDB 的时间                                           |
+| 创建集群     | 30 s         | 2%      | 包含下载 docker image 和启动 PD 的时间                                             |
+| 卷恢复      | 20 s         | 1%      | 包含启动 BR pod 和卷恢复的时间                                                     |
+| 启动 TiKV   | 10~16 min    | 42%     | 包含 TiKV 启动时，启动 rocksdb，读取所有 region 元信息。                             |
+| 数据恢复阶段 | 2～20 min    | 52%      | 包含恢复 raft 共识层数据以及删除 MVCC 数据的时间                                     |
+| 启动 TiDB   | 1 min        | 3%      | 包含下载 docker image 和启动 TiDB 的时间                                          |
 
 > **注意：**
 >
-> 因为卷快照具有崩溃一致性 (Crash Consistency) 的特点，EBS 卷快照恢复阶段 TiKV 启动阶段，需要数据自检和自动修复。经测试，该阶段耗时在 30 分钟以内，如使用高性能盘恢复，如提升 IOPS 和带宽, 额外耗时可缩短到 5 分钟内。使用高性能盘恢复请参考[恢复时间太长（大于 2 小时）](backup-restore-faq.md#恢复时间太长大于-2-小时)。
+> 因为卷快照具有崩溃一致性 (Crash Consistency) 的特点，EBS 卷快照恢复阶段 TiKV 启动阶段和数据恢复阶段，需要初始化数据，即从 S3 下载数据。经测试，该阶段耗时在 30 分钟以内，如使用高性能盘恢复，如提升 IOPS 和带宽, 额外耗时可缩短到 5 分钟内。使用高性能盘恢复请参考[恢复时间太长（大于 2 小时）](backup-restore-faq.md#恢复时间太长大于-2-小时)。
 
 ### 恢复性能数据
 
-卷快照恢复时间取主要决于 TiKV 启动和数据恢复阶段，TiKV 启动受 EBS 卷快照恢复延迟初始化影响，当前 AWS 并没有提供恢复卷延迟初始完成量化指标。根据测试，在 TiDB Operator 推荐 EC2 机型配置下使用 GP3 如下配置，整个恢复时间大致如下：
+卷快照恢复时间取主要决于 TiKV 启动和数据恢复阶段，TiKV 启动和数据恢复都需要读取卷数据。通过快照恢复的卷具有延迟加载的特性，即刚刚恢复完成后不能立刻达到最佳性能，原因是数据必须等数据从 Amazon S3 下载完成后并将其写入到恢复的卷中，后续才能访问这些数据。该延迟加载操作需要一些时间才能完成，并且可能会导致首次访问每个块时的 I/O 操作延迟大大提高。受 EBS 卷快照恢复卷延迟加载影响，TiKV 启动和数据恢复这两个阶段耗时最长。根据我们的测试，在 TiDB Operator 推荐 EC2 机型配置下使用 GP3 如下配置，整个恢复时间大致如下：
```

Comment:
```suggestion
卷快照恢复时间主要取决于启动 TiKV 和数据恢复消耗的时间，启动 TiKV 和恢复数据都需要读取卷数据。通过快照恢复的卷具有延迟加载的特性，即刚刚恢复完成后，不能立刻达到最佳性能。这是因为，从 Amazon S3 下载完数据后，还需要等数据写入到恢复的卷中，这样，后续才能访问这些数据。该延迟加载操作需要一些时间才能完成，并且可能会导致首次访问每个块时的 I/O 操作延迟大大提高。受 EBS 卷快照恢复卷延迟加载影响，TiKV 启动和数据恢复这两个阶段耗时最长。根据测试，在 TiDB Operator 推荐 EC2 机型配置下使用 GP3 如下配置，整个恢复时间大致如下：
```

Reply:
fixed in https://github.com/pingcap/docs-tidb-operator/pull/2169/commits/e34683931ef9c4c531dfdc857f45a2597b396a44